moped-editor/src/components/GridTable/Filters.js

```diff
@@ -511,7 +502,12 @@ const Filters = ({
                     ) ? (
                       <Autocomplete
                         value={value || null}
-                        options={dedupedOptions}
+                        options={autocompleteOptions[fieldName]}
+                        // allow freeSolo option when the operator is contains and contains is not its only operator
+                        freeSolo={
+                          operator === "string_contains_case_insensitive" &&
+                          showFreeSolo
```

Comment:
i like Rose's comment about wanting more info about how this option is used. Moving this logic into a helper function could be helpful to doc and extend the logic in the future. I could see entering an arbitrary string helping with the `string_contains_not_case_insensitive` operator too.